jcli/src/jcli_lib/key.rs

```diff
@@ -2,9 +2,9 @@ use crate::jcli_lib::utils::io;
 use crate::jcli_lib::utils::output_file::{self, OutputFile};
 use bech32::{self, u5, FromBase32, ToBase32};
 use chain_crypto::{
-    bech32::Bech32 as _, AsymmetricKey, AsymmetricPublicKey, Curve25519_2HashDh, Ed25519,
-    Ed25519Bip32, Ed25519Extended, SecretKey, SigningAlgorithm, SumEd25519_12, Verification,
-    VerificationAlgorithm,
+    bech32::Bech32 as _, AsymmetricKey, AsymmetricPublicKey, Ed25519, Ed25519Bip32,
+    Ed25519Extended, EllipticCurve2hashDh, SecretKey, SigningAlgorithm, SumEd25519_12,
```

Comment:
So now we have explicitly selected algorithms for most cryptographic things in jormungandr, but a feature-selected alias for the ECC which affects both the vote protocol and Praos.
This diverges from the general approach of providing generic crypto APIs in the libraries and selecting specific implementations in jormungandr (`blockcfg` is where it should be concentrated, actually).

Can we export both ECC backends in chain-libs unaliased, so that jormungandr can select one of them explicitly? This will remove the issue with being unable to build the library with `--all-features` because of the conflicting backends.